tests/e2e/commands/network.go

```diff
@@ -81,11 +81,30 @@ func StartNetworkWithVersion(version string) string {
 }
 
 /* #nosec G204 */
-func StopNetwork() {
+func StopNetwork(stopCmdFlags ...string) {
+	stopCmdFlasg := append([]string{
+		NetworkCmd,
+		"stop",
+		"--" + constants.SkipUpdateFlag,
+	}, stopCmdFlags...)
+	cmd := exec.Command(
+		CLIBinary,
+		stopCmdFlasg...,
+	)
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		fmt.Println(cmd.String())
+		fmt.Println(string(output))
+		utils.PrintStdErr(err)
+	}
+	gomega.Expect(err).Should(gomega.BeNil())
```

Comment:
really this does not give possibility to check stop error conditions

Reply:
addressed - now `err` is returned